.github/workflows/release-docker.yml

```diff
@@ -20,7 +20,8 @@ jobs:
     steps:
       - name: Checkout tagged commit
         uses: actions/checkout@v2.4.0
-      # TODO: checkout the tag and not master
+        with:
+          ref: ${{ github.event.inputs.realease_tag }}
```

Comment:
Typo!

```suggestion
ref: ${{ github.event.inputs.release_tag }}
```

Reply:
Same typo is also in the action input parameters 🙈

Reply:
Let's fix it everywhere then!

Reply:
done and force pushed